Dockerfile_build_ubuntu64_go1.13

```diff
@@ -24,7 +24,7 @@ RUN apt-get update && DEBIAN_FRONTEND=noninteractive apt-get install -y \
 
 RUN gem install fpm
 
-RUN curl --proto '=https' --tlsv1.2 -sSf https://sh.rustup.rs | sh -s -- -y
+RUN curl --proto '=https' --tlsv1.2 -sSf https://sh.rustup.rs | sh -s -- --default-toolchain stable -y
```

Comment:
We should pull a stable rust version. Ideally we would either save the builder Dockerfile, or hard-code a rust version, but that's for the future.